lib/hydra/works/models/concerns/file.rb

```diff
@@ -0,0 +1,12 @@
+module Hydra::Works
+  # Namespace for Modules with functionality specific to Files
+  module File
```

Comment:
The equivalent namespace in sufia was Sufia::GenericFile, but in hydra-works, Hydra::Works::GenericFile is a class so we can't also use it as a namespace. 

One option: name the class Hydra::Works::GenericFileBase, which would allow us to call this module Hydra::Works::GenericFile instead of Hydra::Works::File.